README.md

```diff
@@ -424,6 +424,93 @@ The local key helper is responsible for:
 - Producing signatures with the binding key
 - Cleanup of the binding key and its artifacts (when the user clears the browser session or the key is unused for a long time)
 
+### Local Key Helper on Android
+On android platform a Local Key Helper is a protocol interface (`ILocalKeyHelper.java`), and its implementation (`LocalKeyHelperImpl.java`) is yet to be determined.
+
+Local KeyHelpers on the Android platform are implemented using [content providers](https://developer.android.com/guide/topics/providers/content-providers). These content providers allow browser applications, such as Chrome, to interact with them by querying and accessing specific APIs defined by the Local KeyHelpers.
+
+#### Content Provider Authority
+Each Local KeyHelper defines a content provider with a authority such as `com.contoso.localkeyhelper`. Different APIs are implemented as separate paths on this content provider.
+
+#### Browser Interaction with Local KeyHelpers
+To interact with Local KeyHelpers, browser applications must define a `<provider>` inside the `<queries>` tag in their manifest file with the same authority `com.contoso.localkeyhelper`. This allows the browser to query the Local KeyHelpers.
+
+##### Example Manifest Entry
+```xml
+<queries>
+    <provider android:authorities="com.contoso.localkeyhelper" />
+</queries>
+```   
+To get the installed applications that provide a content provider with authority "com.contoso.localkeyhelper" and select a specific package "com.contoso.localkeyhelper.entra" to query the content provider, you can use the following Java code:
+
+```java
+PackageManager packageManager = getPackageManager();
+List<PackageInfo> installedPackages = packageManager.getInstalledPackages(PackageManager.GET_PROVIDERS);
```

Comment:
Will it allow Chrome to query all possible content providers on the device? Do they need any additional permissions for that?

Reply:
Chrome needs to declare the <query> tag for package visibility for localkeyhelper provider

Reply:
A specific tag, or a generic tag that allows it to discover all possible providers on the device?

Reply:
I removed the browser need to define the query. Once the local keyhelper is registers with Browser, the Browser can discover it.